src/internal/analytics/__tests__/hooks/use-funnel.test.tsx

```diff
@@ -0,0 +1,129 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { act } from '@testing-library/react';
+import { renderHook } from '../../../../__tests__/render-hook';
```

Comment:
Still renderHook used. In the ideal state I expect to refactor all existing uses of this hook and remove the helper.

If you want to know more details than https://github.com/cloudscape-design/components/pull/1110#discussion_r1200844870 let me know in the DM

Reply:
Refactored 👍🏻